Editor/Resources/EditorWindow/EditorMenu.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using AmazonGameLift.Editor;
+using Editor.Window;
+using UnityEditor;
+using UnityEngine;
+
+namespace Editor.Resources.EditorWindow
+{
+    internal static class EditorMenu
+    {
+        private static GameLiftPlugin GetPluginWindow()
+        {
+            var inspectorType = Type.GetType("UnityEditor.GameView,UnityEditor.dll");
+            var window = UnityEditor.EditorWindow.GetWindow<GameLiftPlugin>(inspectorType);
+            window.titleContent = new GUIContent("Amazon GameLift", window._icon);
+            return window;
+        }
+
+        [MenuItem("Amazon GameLift/Show Amazon GameLift Window", priority = 0)]
+        public static void ShowWindow()
+        {
+            GetPluginWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Bring Panel to Front", priority = 1)]
+        public static void FocusPanel()
+        {
+            ShowWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Set AWS Account Profiles", priority = 100)]
+        public static void OpenAccountProfilesTab()
+        {
+            GetPluginWindow().OpenTab("Credentials");
```

Comment:
Please don't use hard strings here. I recommend switching the inner class/list of constants to an enum, but regardless you should use that inner class here

Reply:
Swapped to enum